### ISSUE TITLE
Issues with package import

### ISSUE THREAD

**Issue by kjawadDeveloper2** [tendermint/flutter#173]
**Desription**
After upgrading to flutter 2.8.0. I am getting the following issue. 

Because every version of transaction_signing_gateway from git depends on cosmos_utils from git {relative: false, url: https://github.com/tendermint/flutter.git, ref: starport-redesign, path: packages/cosmos_utils} and pylons_wallet depends on cosmos_utils from git {relative: false, url: https://github.com/tendermint/flutter.git, ref: main, path: packages/cosmos_utils}, transaction_signing_gateway from git is forbidden.

**Comment by kjawadDeveloper2** [comment]
For anyone else who is facing this issue use the following imports.
```
 cosmos_ui_components:
    git:
      url: https://github.com/tendermint/flutter.git
      path: packages/cosmos_ui_components
      ref: starport-redesign

  cosmos_utils:
    git:
      url: https://github.com/tendermint/flutter.git
      path: packages/cosmos_utils
      ref: starport-redesign

  transaction_signing_gateway:
    git:
      url: https://github.com/tendermint/flutter.git
      path: packages/transaction_signing_gateway
      ref: starport-redesign
```
And make sure that the flutter_svg is of latest version.

**Comment by andrzejchm** [comment]
this will be fixed with v2.0.1 release
